docs/changelog-fragments/621.bugfix.rst

```diff
@@ -0,0 +1 @@
+Downloading files larger thank 64kB over SCP no longer fails. -- by :user:`Jakuje`
```

Comment:
The trailing period should be after the byline, if it continues a sentence. Otherwise, a byline w/o a trailing period should go to a separate line with an empty-line separator.
```suggestion
Downloading files larger thank 64kB over SCP no longer fails -- by :user:`Jakuje`.
```